content/workshops/cascading-layouts.md

```diff
@@ -1,12 +1,11 @@
 ---
 title: Cascading Layouts
-sub: A workshop on resilient CSS layouts, June 24-26, 2024
-feature: workshop
+sub: A workshop on resilient CSS layouts
 author: miriam
-date: 2024-06-24
+date: 2024-12-31
```

Comment:
Ensures that this workshop appears first in the list, but without a "2025 Posts" header on Miriam's author page.